share/config.c

```diff
@@ -183,10 +183,10 @@ static struct
     { &CONFIG_JOYSTICK_BUTTON_L2,     "joystick_button_l2",     6 },
     { &CONFIG_JOYSTICK_BUTTON_R2,     "joystick_button_r2",     7 },
     { &CONFIG_JOYSTICK_BUTTON_START,  "joystick_button_start",  8 },
-    { &CONFIG_JOYSTICK_DPAD_L,        "joystick_dpad_l",       -1 },
-    { &CONFIG_JOYSTICK_DPAD_R,        "joystick_dpad_r",       -1 },
-    { &CONFIG_JOYSTICK_DPAD_U,        "joystick_dpad_u",       -1 },
-    { &CONFIG_JOYSTICK_DPAD_D,        "joystick_dpad_d",       -1 },
+    { &CONFIG_JOYSTICK_DPAD_L,        "joystick_dpad_l",        9 },
+    { &CONFIG_JOYSTICK_DPAD_R,        "joystick_dpad_r",        10 },
+    { &CONFIG_JOYSTICK_DPAD_U,        "joystick_dpad_u",        11 },
+    { &CONFIG_JOYSTICK_DPAD_D,        "joystick_dpad_d",        12 },
```

Comment:
Well, I chose those since they were the next four in the list 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12. I can change it back if you want. I'm fairly new to this codebase, so I don't know what's the appropriate thing to do. I just thought it would save the end user a bit of hassle, since the rest of the Wiimote functionality except the D-pad works out of the box as per documentation (manual.txt), and manual.txt doesn't mention anything about having to change the joystick_dpad_* values. Then again, people using gamepads are probably also going to fiddle with neverballrc, anyway, since SDL is probably not going to guess their setup correctly. Any suggestions?